Fix LdapConfig

Fix runtime error:
```
	Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [javax.servlet.Filter]: Factory method 'springSecurityFilterChain' threw exception; nested exception is scala.MatchError: (***********.COM,auth.ad.domain) (of class scala.Tuple2)
		at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
		at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:579)
		... 67 more
	Caused by: scala.MatchError: (************.COM,auth.ad.domain) (of class scala.Tuple2)
		at za.co.absa.hyperdrive.trigger.api.rest.auth.LdapAuthentication$$anonfun$validateParams$1.apply(LdapAuthentication.scala:46)
		at za.co.absa.hyperdrive.trigger.api.rest.auth.LdapAuthentication$$anonfun$validateParams$1.apply(LdapAuthentication.scala:46)
		at scala.collection.immutable.List.foreach(List.scala:392)
		at za.co.absa.hyperdrive.trigger.api.rest.auth.LdapAuthentication.validateParams(LdapAuthentication.scala:46)
		at za.co.absa.hyperdrive.trigger.api.rest.auth.LdapAuthentication.configure(LdapAuthentication.scala:53)
		at za.co.absa.hyperdrive.trigger.api.rest.WebSecurityConfig$ApiWebSecurityConfigurationAdapter.configure(WebSecurityConfig.scala:110)

```

--- a/src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/auth/LdapAuthentication.scala
+++ b/src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/auth/LdapAuthentication.scala
@@ -46,6 +46,7 @@ class LdapAuthentication extends HyperdriverAuthentication {
     requiredParameters.foreach {
       case param if param._1.isEmpty =>
         throw new IllegalArgumentException(s"${param._2} has to be configured in order to use ldap authentication")
+      case _ => 
     }
   }
 

